tornado/web.py

```diff
@@ -139,11 +139,19 @@ def get(self):
 DEFAULT_SIGNED_VALUE_MIN_VERSION = 1
 """The oldest signed value accepted by `.RequestHandler.get_secure_cookie`.
 
-May be overrided by passing a ``min_version`` keyword argument.
+May be overridden by passing a ``min_version`` keyword argument.
 
 .. versionadded:: 3.2.1
 """
 
+DEFAULT_SIGN_KEY_VERSION = 0
```

Comment:
I think I'd just get rid of this and require that when a key dict is used an explicit version is specified (you already do this in set_secure_cookie; it looks like this constant is only used when calling create_signed_value directly)